helm_deploy/prisoner-content-hub-frontend/values.yaml

```diff
@@ -1,8 +1,8 @@
 nameOverride: ""
 fullnameOverride: ""
 
-replicaCount: 2
-maxReplicaCount: 3
+replicaCount: 3
+maxReplicaCount: 6
```

Comment:
Up for discussion on these....

Reply:
I'm keen to have this driven by the results of the performance test. Part of me wonders if we'd actually cope with fairly high load with a small number of pods seeing as the node process is fairly lightweight & fast, but we can definitely make better decisions with some data. 

We should be able to say "adding 1 pod to the frontend increases capacity by `m` users" (approx, and with all the caveats) and judge how & when to add them. It feels like we need a "brave new world" gif 🤪.

Reply:
Yeah it's a difficult one , we could set to 6 as that's what we've got now but those 6 will be shared between all 3 sites.

Reply:
See Lucas' notes on what we managed with testing. 3 will be fine for now.